pkg/router/router_test.go

```diff
@@ -842,6 +911,23 @@ func (e mustCreateRoute) Apply(h *harness) error {
 	if e.targetServiceName != "" {
 		serviceName = e.targetServiceName
 	}
+	weight := new(int32)
```

Comment:
Not a big deal, but is there a reason you use `weight := new(int32)` instead of using `var weight int32`?

Reply:
No, I was just inspired by the previous code which did this: `Weight: new(int32)`.